### PR TITLE
feat: :art: Add Kebbi

### PR DESCRIPTION
**Pull request by RaxCruz**
## Description

[5930](https://redmine.kingkit.codes/issues/5930)

## Motivation and Context

切版- Kebbi 車子

## Screenshots (if appropriate)
kebbi (1)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: my-app/src/App.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.App {
+    
+}
```
Comment (shengyuan) [inline my-app/src/App.css]: 無效的程式，建議先不加入。

File: my-app/src/App.tsx [hunk 2]
```diff
@@ -1,10 +1,60 @@
-import React from "react";
+import React, { useRef, useState } from "react";
 import "./App.css";
-
+import { ReactComponent as Circle } from "./images/circle.svg";
+import { ReactComponent as AiCar1 } from "./images/aicar1.svg";
+import { ReactComponent as AiCar2 } from "./images/aicar2.svg";
 function App() {
+  const KebbiRef: any = useRef(null);
```
Comment (shengyuan) [inline my-app/src/App.tsx]: 好奇這裡寫 `any` 的效果是？如果省略不寫，會發生錯誤嗎？

File: my-app/src/App.tsx [hunk 3]
```diff
@@ -1,10 +1,60 @@
...
+  const [isKebbiClick, setIsKebbiClick] = useState(false);
+  const [kebbiOriginPosition, setKebbiOriginPosition] = useState({
+    left: 0,
+    top: 0,
+  });
+  const [kebbiPosition, setKebbiPosition] = useState({
+    left: 0,
+    top: 0,
+  });
+  const mousemoveHandler = (e: any) => {
```
Comment (shengyuan) [inline my-app/src/App.tsx]: event object，React 有型別的支持，因此這裡的型別應該可以宣告，而不是 any。
其他專案的範例，
image

File: my-app/src/App.tsx [hunk 4]
```diff
@@ -1,10 +1,60 @@
-import React from "react";
+import React, { useRef, useState } from "react";
 import "./App.css";
-
+import { ReactComponent as Circle } from "./images/circle.svg";
+import { ReactComponent as AiCar1 } from "./images/aicar1.svg";
+import { ReactComponent as AiCar2 } from "./images/aicar2.svg";
 function App() {
+  const KebbiRef: any = useRef(null);
+  const [isKebbiClick, setIsKebbiClick] = useState(false);
```
Comment (shengyuan) [inline my-app/src/App.tsx]: 建議可以調整一下命名，例：`[inDragging, setInDragging]`。
來表達小車拖拉的狀態。

Comment (RaxCruz) [reply my-app/src/App.tsx]: 已調整

File: my-app/src/App.tsx (hunk 3 above)
Comment (RaxCruz) [reply my-app/src/App.tsx]: 已調整

File: my-app/src/App.tsx (hunk 2 above)
Comment (RaxCruz) [reply my-app/src/App.tsx]: 省略不會發生錯誤 已調整

File: my-app/src/App.css (hunk 1 above)
Comment (RaxCruz) [reply my-app/src/App.css]: 已調整

File: my-app/src/App.tsx [hunk 5]
```diff
@@ -1,10 +1,60 @@
...
+        top: e.pageY - kebbiOriginPosition.top,
+      });
+
+    }
+  };
+  const mouseupHandler = () => {
+    setKebbiPosition({ left: 0, top: 0 });
+    setInDragging(false);
+  };
+  const mousedownHandler = (e: any) => {
```
Comment (shengyuan) [inline my-app/src/App.tsx:30]: React 有支持事件的 typescript，請再調整一下。

**Pull request closed without merging**